package.json

```diff
@@ -9,19 +9,14 @@
     "format:base": "prettier \"./**/*.{ts,js}\"",
     "format:fix": "npm run format:base -- --write",
     "format:check": "npm run format:base -- --check",
-    "test": "nyc mocha --timeout=30000",
-    "test:unit": "mocha --exclude test/git.spec.js",
-    "coverage": "nyc report --reporter=lcov",
+    "test": "jest --verbose --silent",
+    "test:unit": "jest --verbose --silent --testPathIgnorePatterns test/git.integration-test.js",
+    "test:git-integration": "jest --verbose --silent --testPathPattern test/git.integration-test.js",
     "release": "bin/cli.js"
   },
-  "nyc": {
-    "exclude": [
-      "tmp/**"
-    ]
-  },
   "repository": "absolute-version/commit-and-tag-version",
   "engines": {
-    "node": ">=14"
+    "node": ">=16 <=20"
```

Comment:
Is there an advantage to putting in an upper limit?

Reply:
Mainly I put this in because with the unit test suite we are able to guarantee it works with 18 and 20 currently

Reply:
Mainly I put this in because with the unit test suite we are able to guarantee it works with 18 and 20 currently, happy to remove the upper limit